### ISSUE TITLE
Question about "canonical reproducible form"

### ISSUE THREAD

**Issue by jez** [google/cargo-raze#154]
Are there any recommendations about how to deal with DEBUG messages like these?

```
DEBUG: Rule 'raze__lscolors__0_7_0' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "1f77452267149eac960ded529fe5f5460ddf792845a1d71b5d0cfcee5642e47e"
DEBUG: Repository raze__lscolors__0_7_0 instantiated at:
  no stack (--record_rule_instantiation_callstack not enabled)
Repository rule http_archive defined at:
  /private/var/tmp/_bazel_jez/a5bbbd46a58ce45d1f576b6b0830c3ec/external/bazel_tools/tools/build_defs/repo/http.bzl:336:16: in <toplevel>
DEBUG: Rule 'raze__ansi_term__0_12_1' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "d52a9bb7ec0cf484c551830a7ce27bd20d67eac647e1befb56b0be4ee39a55d2"
DEBUG: Repository raze__ansi_term__0_12_1 instantiated at:
  no stack (--record_rule_instantiation_callstack not enabled)
Repository rule http_archive defined at:
  /private/var/tmp/_bazel_jez/a5bbbd46a58ce45d1f576b6b0830c3ec/external/bazel_tools/tools/build_defs/repo/http.bzl:336:16: in <toplevel>
```

I'm using this config:

```toml
[raze]
workspace_path = "//third_party/cargo"
target = "x86_64-apple-darwin"
genmode = "Remote"
```

I can get rid of them by manually inserting the the `sha256` hashes into the `crates.bzl` file manually, and afaik the only problem there is that the next time I run `cargo raze` I'll have to go back and re-add them, which is fine.

Is there some `cargo raze` invocation I'm missing that will insert those additional lines for me? Just want to make sure I'm not missing something.

Anyways, just wanted to say thanks for a great project. Everything just worked on the first try which is almost never the case when I do anything Bazel related so you all get a huge 🏆 in my book!

**Comment by kornholi** [comment]
This will be fixed by #149.

**Comment by jez** [comment]
Oh great thanks!
